src/screens/CreatePost/index.js

```diff
@@ -833,7 +813,6 @@ const CreatePost = () => {
             topics={listTopic}
             chatTopics={listTopicChat}
             onClose={() => sheetTopicRef.current.close()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The removal of the `saveOnClose` prop from the `SheetTopic` component simplifies the code and makes it easier to understand. However, make sure that the removed prop is not needed anymore or has been handled elsewhere.
```diff
- // saveOnClose={(v, chatTopic) => onSaveTopic(v, chatTopic)}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->